index.html

```diff
@@ -966,7 +966,8 @@ <h2><dfn>Coalesced events</dfn></h2>
             <p>A <a>PointerEvent</a> has an associated <dfn>coalesced events list</dfn> (a list of
             zero or more <code>PointerEvent</code>s). For trusted <code>pointermove</code> and
             <code>pointerrawupdate</code> events, the list is a sequence of all <code>PointerEvent</code>s
-            that were coalesced into this event. For all other trusted event types, it is an empty list.
+            that were coalesced into this event, or if no events were coalesced, it contains a clone of the "parent" event itself.
```

Comment:
From call just now, idea from @flackr Right after this line, something along the lines of: the parent event is the result of all these events, with a hint that UA may resample or otherwise average/modify